.gitignore

```
@@ -141,6 +141,7 @@ dmypy.json
 *~
 /charmcraft/_version.py
 /results/
+.*.swp
```

Hey while you're at it could you do:

```suggestion
.*.*swp
```

This gets other swap files like `.kate-swp` too :-)